agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -67,11 +67,11 @@ const (
 	// TODO: Benchmark and re-evaluate this value
 	LoadImageTimeout        = 10 * time.Minute
 	pullImageTimeout        = 2 * time.Hour
-	createContainerTimeout  = 4 * time.Minute
-	stopContainerTimeout    = 30 * time.Second
-	removeContainerTimeout  = 5 * time.Minute
-	inspectContainerTimeout = 30 * time.Second
-	removeImageTimeout      = 3 * time.Minute
+	CreateContainerTimeout  = 4 * time.Minute
```

Comment:
lint: Can you add documentation for all of the globally visible const's? See https://github.com/golang/go/wiki/CodeReviewComments#doc-comments and https://golang.org/doc/effective_go.html#commentary for more details.